examples/README.md

```diff
@@ -0,0 +1,30 @@
+# Express examples
+
+This page contains list of examples using Express.
+
+- Authentication with login and passowrd - [examples/auth](./auth)
+- HTTP content negotiation - [examples/content-negotiation](./content-negotiation)
+- Working with cookie-based session - [examples/cookie-sessions](./cookie-sessions)
+- Working with cookies via [`cookie-parser`](https://github.com/expressjs/cookie-parser) - [examples/cookies](./cookies)
+- Files transfer to client - [examples/downloads](./downloads)
+- Working with Embedded JavaScript templating (ejs) - [examples/ejs](./ejs)
+- Example pages for HTTP error - [examples/error-pages](./error-pages)
```

Comment:
This description also reads weird, including the part where this list of examples, I think "Example" is probably implied. What about something like "Creating error pages"?